js/ui/overlay/ui.overlay.js

```diff
@@ -257,9 +259,15 @@ const Overlay = Widget.inherit({
         return this._$content;
     },
 
-    _init: function() {
+    _setDeprecatedOptions() {
         this.callBase();
+        extend(this._deprecatedOptions, {
+            'elementAttr': { since: '21.2', message: 'Use the \'wrapperAttr\' option instead' }
```

Comment:
```suggestion
            'elementAttr': { since: '21.2', message: 'Use the "wrapperAttr" option instead' }
```

Reply:
I think we should add smth like "This property is deprecated ...". Let's discuss it with technical writers

Reply:
done